susemanager-frontend/susemanager-nodejs-sdk-devel/package.json

```diff
@@ -34,8 +35,10 @@
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-react-hooks": "^1.0.2",
     "flow-bin": "^0.107.0",
-    "jest": "^24.7.1",
+    "jest": "^26.6.3",
+    "jquery": "3.1.0",
```

Comment:
This dependency is actually available for the deployed page itself through `susemanager-frontend-libs`, but the test pipeline doesn't know anything about it. This overlapping definition should be resolved by https://github.com/SUSE/spacewalk/issues/13145.

Reply:
So, if I understand it correctly, until #13145 is addressed we will have two places where we fetch and import the jQuery lib, is this correct? If so, I'd better drop the jQuery lib from `susemanager-frontend-libs` early right now in the time context of this PR.

Reply:
If I were to move one dependency over from `frontend-libs` to `nodejs-sdk-devel`, it would make sense to already move all of them over, no? Or are there any downsides that I'm missing here? Perhaps also ping @juliogonzalez, please see https://github.com/SUSE/spacewalk/issues/13145 for context.

Reply:
In the end, as agreed in the discussion we had offline, it's okay to keep the double of the dependency for the moment, as we plan to accomplish #13145 soon, it is harmless in the end to wait until we tackle them all together at the same time. :+1:

Reply:
Well, I have one question, but will write it at the other card.